great_expectations/data_context/types/base.py

```diff
@@ -558,6 +558,17 @@ def validate_schema(self, data, **kwargs):
 continue.
                     """
             )
+        if "gcs_options" in data and data["class_name"] in [
+            "InferredAssetGCSDataConnector",
+            "ConfiguredAssetGCSDataConnector",
+        ]:
+            azure_options = data["gcs_options"]
```

Comment:
var should be `gcs_options` (here and below)

Reply:
Ah good call! Changed 😄